cl_manycore/libraries/bsg_manycore_driver.h

```diff
@@ -144,9 +144,13 @@ int hb_mc_all_host_req_complete(uint8_t fd);
 int hb_mc_get_config(uint8_t fd, hb_mc_config_id_t id, uint32_t *cfg);
 uint8_t hb_mc_get_manycore_dimension_x();
 uint8_t hb_mc_get_manycore_dimension_y();
+uint8_t hb_mc_get_host_intf_coord_x();
```

Comment:
These methods are provided by the config API. Is there a reason you didn't use those?

Reply:
They are actually deprecated, I've already replaced them with the methods in config API. Should I delete them or leave them with a depreacted tag? (leaving them with a deprecated tag generates a lot of warnings on compile, too messy in my opinion)

Reply:
Deprecated tag.